### PR TITLE
1436790/1438462: Part 16: 20 files with MSDN link fixes.

### PR DESCRIPTION
**Pull request by DennisLee-DennisLee**
Contributes to PR #[2037](https://github.com/dotnet/docs/issues/2037) and applies to the following work items:
- [User Story 1436790: Content Update: update MSDN links for .NET content](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1436790)
- [Task 1438462: Update MSDN links for .NET content](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1438462)

### CODE REVIEW

File: docs/framework/data/wcf/data-service-versioning-wcf-data-services.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ The [!INCLUDE[ssODataFull](../../../../includes/ssodatafull-md.md)] enables you
...
+ By default, WCF Data Services uses version 1.1 of CSDL to represent a data model. This is always the case for data models that are based on either a reflection provider or a custom data service provider. However, when the data model is defined by using the [!INCLUDE[adonet_ef](../../../../includes/adonet-ef-md.md)], the version of CSDL returned is the same as the version that is used by the [!INCLUDE[adonet_ef](../../../../includes/adonet-ef-md.md)]. The version of the CSDL is determined by the namespace of the [Schema element (CSDL)](/ef/ef6/modeling/designer/advanced/edmx/csdl-spec#schema-element-csdl). For more information, see the specification [\[MC-CSDL\]: Conceptual Schema Definition File Format](https://go.microsoft.com/fwlink/?LinkId=159072).
```
Comment (mairaw) [inline docs/framework/data/wcf/data-service-versioning-wcf-data-services.md]:
```suggestion
 By default, WCF Data Services uses version 1.1 of CSDL to represent a data model. This is always the case for data models that are based on either a reflection provider or a custom data service provider. However, when the data model is defined by using the [!INCLUDE[adonet_ef](../../../../includes/adonet-ef-md.md)], the version of CSDL returned is the same as the version that is used by the [!INCLUDE[adonet_ef](../../../../includes/adonet-ef-md.md)]. The version of the CSDL is determined by the namespace of the [Schema element (CSDL)](/ef/ef6/modeling/designer/advanced/edmx/csdl-spec#schema-element-csdl).
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @DennisLee-DennisLee. A few comments before we can merge.

**Comment by DennisLee-DennisLee** [comment]
#sign-off

**Review by mairaw — APPROVED** [review]
Looks good! Thanks @DennisLee-DennisLee ❄️

**Pull request merged**